docs/src/clusters.md

```diff
@@ -74,20 +74,21 @@ solana config set --url https://testnet.solana.com
 
 ##### Example `solana-validator` command-line
 
+[comment]: <> (UPDATE ci/live-cluster-sanity.sh TOO!!)
```

Comment:
TIL: https://stackoverflow.com/a/20885980

Reply:
I noticed docusaus can't handle this correctly, this is the reason of failing travis build.

Reply:
this is now fixed.